Lombiq.HelpfulLibraries.LinqToDb/Extensions/CustomSqlExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using LinqToDB;
+using System;
+
+namespace Lombiq.HelpfulLibraries.LinqToDb.Extensions
+{
+    public static class CustomSqlExtensions
+    {
+        [Sql.Expression(ProviderName.SqlServer, "JSON_VALUE({0}, {1})", ServerSideOnly = true, InlineParameters = true)]
+        [Sql.Expression(ProviderName.SQLite, "json_extract({0}, {1})", ServerSideOnly = true, InlineParameters = true)]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA1801:Review unused parameters", Justification = "Used in the Sql.Expression attribute.")]
+        public static string JsonValue(object expression, string path) => throw new InvalidOperationException();

```

Comment:
Miért mindegyik exceptiont dob?

Reply:
Mert nem történik ott semmilyen megvalósítás, mert nem olyan bonyolult a kifejezés, csak behelyettesítés azonnal az `Sql.Expression`-be, és azt adja vissza meghívás esetén. 
Pár példa erre: [https://github.com/linq2db/linq2db/blob/master/Tests/Linq/Linq/ExpressionsTests.cs](https://github.com/linq2db/linq2db/blob/master/Tests/Linq/Linq/ExpressionsTests.cs)

Reply:
Aha, akkor írd le ide légyszi, hogy itt nincs szükség implementációra. Másrészt egy fokkal talán jobb lenne, ha inkább `null`-t adnának vissza, egy fokkal kevésbé fura.